documentation/guide-for-adding-banner-ads.md

```diff
@@ -0,0 +1,80 @@
+# Banner Ads Implementation Guide for all joomla.org sites
+This file describes how Joomla.org web masters should implement the ad serving code on our web properties.
+
+## ads.joomla.org
+We are running our own internal ([Revive](https://www.revive-adserver.com/)) ad server that should be serving all our ads. *We should not be serving ads directly from AdSense.*
+
+## Ad code
+The code to show ads is simple and in two parts.
+
+### General script
+```html
+<script async src="//ads.joomla.org/www/delivery/asyncjs.php"></script>
+```
+This code should be included **once** per page. The best way to include this code is most likely by pasting it into the template of the site. The code should be inside the `<body>` of the page near the `</body>` closing tag.
+
+### Zone specific code
+```html
+<ins data-revive-zoneid="xx" data-revive-id="4bacaeba2f8655edc9ca810c946aab5a"></ins>
+```
+The xx should be replaced with a zone ID given to you by the team managing the ad server. 
+
+A good way to do add this code is to create a Joomla module with custom html. Make sure you disable the editor when pasting in or editing this code as it might otherwise strip some of the html.
+
+If you need to get one or more zone id's for a site then please contact soren.beck-jensen@community.joomla.org
+
+## Ad formats
+We currently have two (well kind of three) ad formats that you can implement on your site.
+
+### 728 x 90 (Leaderboard)
+This ad format is the most popular online and we should aim to have at least one of these on every page. Placing it above, below or in the middle of content are all good options.
+
+![Eample banner](http://placehold.it/728x90)
+
+### 300 x 250 (Medium rectangle)
+This ad format is also very popular and generally attracts more clicks, but is currently not easy to fit into our generic template (*See below for solution*). 
+
+![Eample banner](http://placehold.it/300x250)
+
+### 270 x 225 (Custom size)
+The width of the right hand column the current standard Joomla template is only 270px wide so fitting the above banner into breaks layout. To overcome this, we have developed a little code snippit that will scale the above medium rectangle down a little.
```

Comment:
The side column widths are configurable in the template's settings.  The template uses a default 3-6-3 configuration if you use all three columns (most sites are in a 9-3 configuration for practical purposes).  Instead of needing extra CSS, I'd suggest having the site use a 8-4 configuration (at the 980px viewport the side column's 300 pixels and bigger at the next size up, so it'll fit comfortably on 2 of the 3 viewports that support columns).